lib/cli/kit/report_errors.rb

```diff
@@ -0,0 +1,58 @@
+require 'English'
+
+module CLI
+  module Kit
+    module ReportErrors
+      class << self
+        attr_accessor :exception
+      end
+
+      # error_reporter should support the interface:
+      #  .call(notify_with, logs = nil, stage = nil, api_key = nil, custom_metadata = {})
+      def self.setup(logfile_path, error_reporter)
```

Comment:
Going back and forth on whether to actually bring in bugsnag or leave it at this kind of awkward integration level.

Reply:
What would an app need to implement to use ReportErrors?

Reply:
I made the comment more useful.